### ISSUE TITLE
Maintenance

### ISSUE THREAD

**Issue by Bachstelze** [kariminf/LangPi#1]
Is this project still under development?

**Comment by kariminf** [comment]
Yes, there are a lot to be improved. The project was first a part of AllSummarizer, then I split it to be a library for NLP tasks.
But, for now, I'm busy with my PhD and September I will start teaching. So, I have less time to focus on it.
